lib/feature_management.rb

```diff
@@ -138,6 +138,15 @@ def self.proofing_device_profiling_decisioning_enabled?
     end
   end
 
+  def self.idv_send_issue_and_expiration_dates_to_aamva?
+    case IdentityConfig.store.aamva_issue_and_expiration_date_validation
+    when :log_only then true
+    when :disabled then false
```

Comment:
Since this can only be two states, would a boolean make more sense?

An example in some of our other "track only" configurations for RackAttack (though I think "log only" might make more sense for this context):

https://github.com/18F/identity-idp/blob/4c26a3c33c0282cf620dc1bbbd93b0e1a45e6c2b/config/application.yml.default#L105

https://github.com/18F/identity-idp/blob/4c26a3c33c0282cf620dc1bbbd93b0e1a45e6c2b/config/application.yml.default#L164

https://github.com/18F/identity-idp/blob/4c26a3c33c0282cf620dc1bbbd93b0e1a45e6c2b/config/application.yml.default#L230

https://github.com/18F/identity-idp/blob/4c26a3c33c0282cf620dc1bbbd93b0e1a45e6c2b/config/application.yml.default#L248

Reply:
I understand that we added it as a symbol since we are considering adding an `enabled` state. If we do make the decision to enable this is there a reason for keeping this flag with these states? Do we foresee a situation where we need to quickly revert to a log only state or a disabled state using config?

Reply:
Ah, wasn't aware of the intention to add a third value here. To your question, I suppose that's ultimately up to y'all, but my guess is probably not?

Although the same questions apply for some of those "track only" RackAttack configurations, which I also feel could probably be removed at this point?

Reply:
I think I will probably switch to the `track_only` pattern, since I don't think there's a big risk in _sending_ the data to AAMVA--the risk is in making decision based on AAMVA's response. I'd like to retain the ability to revert to "track only" mode at least for a little while.

Reply:
Actually, for this PR we don't really need a feature flag--we'll probably have one if/when we start making decisions on AAMVA's responses here.